File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/postgresql/SpecialTypes.scala
package dev.typr.foundations.docs.postgresql
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*


import java.util.{Map as JMap}

@SuppressWarnings(Array("unused"))
object SpecialTypes:
  //start
  val hstoreType: PgType[Map[String, String]] = PgTypes.hstore
  val vectorType: PgType[Vector] = PgTypes.vector
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/postgresql/SpecialTypes.java
package dev.typr.foundations.docs.postgresql;

import dev.typr.foundations.PgType;
import dev.typr.foundations.PgTypes;
import dev.typr.foundations.data.Vector;
import java.util.Map;

@SuppressWarnings("unused")
public class SpecialTypes {
  // start
  PgType<Map<String, String>> hstoreType = PgTypes.hstore;
  PgType<Vector> vectorType = PgTypes.vector;
  // stop
}
